### PR TITLE
Use tech site URL for image

### PR DESCRIPTION
**Pull request by microbit-mark**
Use the eventual URL for the image so it is visible on the tech site and on Github

### CODE REVIEW

File: spec/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ In micro:bit V1 there are UART and SWD signals connecting the Interface MCU (KL2
 The micro:bit V2 adds an internal I2C bus connected to the Interface MCU (KL27), the Target MCU (nRF52), and the motion sensors (in V1 the motions sensors are connected to the external I2C bus, connected only to the Target MCU (nRF51), more info in the [Tech Site I2C page](https://tech.microbit.org/hardware/i2c-shared/)).
 This new I2C bus allows the Interface (KL27) to provide additional features to the  Target (nRF52), and to co-operate to set the board into different power modes (more info in the [Power Management Spec](https://github.com/microbit-foundation/spec-power-management/)).
 
-![I2C Diagram](img/i2c-diagram.png)
+![I2C Diagram](https://tech.micribit.org/software/spec-i2c-protocol/spec/img/i2c-diagram.png)
```
Comment (microbit-carlos) [inline spec/index.md]:
```suggestion
![I2C Diagram](https://tech.microbit.org/software/spec-i2c-protocol/spec/img/i2c-diagram.png)
```

**Review by microbit-carlos — APPROVED** [review]
LGTM 👍 
Thanks Mark!

File: spec/index.md (hunk 1 above)
Comment (microbit-carlos) [reply spec/index.md]: Already updated.

Comment (microbit-mark) [reply spec/index.md]: Thanks Carlos. I caught that in https://github.com/microbit-foundation/spec-i2c-protocol/pull/5/commits/2f49b525ed85f12c32b925f9a5247a1a76fcedef

**Pull request merged**